plugin/plugin.go

```diff
@@ -344,6 +350,36 @@ func (p *plugin) generateIntValidator(variableName string, ccTypeName string, fi
 	}
 }
 
+func (p *plugin) generateLengthValidator(variableName string, ccTypeName string, fieldName string, fv *validator.FieldValidator) {
+	if fv.LengthGt != nil {
+		p.P(`if !( len(`, variableName, `) > `, fv.LengthGt, `) {`)
+		p.In()
+		errorStr := fmt.Sprintf(`be greater than '%d'`, fv.GetLengthGt())
+		p.generateErrorString(variableName, fieldName, errorStr, fv)
```

Comment:
In general you can't use the function here, because it will print something like `value bla must be greater than '%d'`, which doesn't mention the length. You'll need to create a different function for this.